scout/server/blueprints/cases/controllers.py

```diff
@@ -124,8 +124,11 @@ def case(store, institute_obj, case_obj):
                                 .format(hpo_term['phenotype_id']))
 
     # other collaborators than the owner of the case
-    o_collaborators = [store.institute(collab_id) for collab_id in case_obj['collaborators'] if
-                       collab_id != case_obj['owner']]
+    o_collaborators = []
+    for collab_id in case_obj['collaborators']:
+        if collab_id != case_obj['owner'] and store.institute(collab_id):
```

Comment:
Better safe than sorry!  👍

Reply:
That was adding None things to the array of collab_ids, because it was using "Select institite" for ids

Reply:
Yep: when I saw your picture, and recalled Anna clicking "Share" without selecting an institute it was all clear.. 😄